pkg/loop/internal/relayer/pluginprovider/chainreader/chain_reader.go

```diff
@@ -307,6 +306,17 @@ func convertExpressionToProto(expression query.Expression) (*pb.Expression, erro
 	return pbExpression, nil
 }
 
+func confidenceToProto(level primitives.ConfidenceLevel) pb.Confidence {
+	switch level {
```

Comment:
Did you check how `DestOptimisticConfirmations` is used? Does it need its own enum?

Reply:
Yes, it is it's own configuration component, but I get conflicting information on whether this will be included in CCIPv2. Since both endpoints are configurable, I suggest we leave it with the two enums and add a new one in the case it's needed for v2.